<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessingAlgoWidget</class>
 <widget class="QWidget" name="ProcessingAlgoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="lblInfo">
     <property name="text">
      <string>Select either Algorithm or Python Script under &quot;Processing&quot; to perform processing on each chunk of live data.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="MantidQt::MantidWidgets::AlgorithmSelectorWidget" name="algoSelector">
      <property name="showExecuteButton" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="MantidQt::API::AlgorithmPropertiesWidget" name="algoProperties">
      <property name="algorithmName">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="editorContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="toolTip">
           <string>Save the current script to a file</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoad">
          <property name="toolTip">
           <string>Load a file into the current script</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ScriptEditor" name="editor"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScriptEditor</class>
   <extends>QsciScintilla</extends>
   <header>MantidQtWidgets/Common/ScriptEditor.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::AlgorithmSelectorWidget</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/AlgorithmSelectorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::API::AlgorithmPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/AlgorithmPropertiesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
